ext/prism/extconf.rb

```diff
@@ -77,15 +57,38 @@ def make(env, target)
 
 # On non-CRuby we only need the shared library since we'll interface with it
 # through FFI, so we'll build only that and not the C extension. We also avoid
-# `require "mkmf"` as that prepends the LLVM toolchain to PATH on TruffleRuby,
-# but we want to use the native toolchain here since libprism is run natively.
+# `require "mkmf"` as that prepends the GraalVM LLVM toolchain to PATH on TruffleRuby < 24.0,
+# but we want to use the system toolchain here since libprism is run natively.
 if RUBY_ENGINE != "ruby"
   generate_templates
-  make(env, "build/libprism.#{RbConfig::CONFIG["SOEXT"]}")
+  soext = RbConfig::CONFIG["SOEXT"]
```

Comment:
We're no longer requiring `rbconfig` in this file before this block, is that okay?

Reply:
Good catch, I'll move the `require`.
RubyGems requires `rbconfig` IIRC but best to not rely on that, and it could be an issue if someone has `RUBYOPT=--disable-gems` in their environment.

Reply:
Fixed